Tests/test_imageops.py

```diff
@@ -99,6 +102,21 @@ def test_fit_same_ratio():
         assert new_im.size == (1000, 755)
 
 
+def test_contain():
```

Comment:
I think these three cases can be refactored using `pytest.mark.parametrize`?

Reply:
I've pushed a commit for this